README.md

```diff
@@ -1,3 +1,280 @@
-Placeholder for possible webpack loader for Twitter Bootstrap
+# bootstrap-loader
 
-Considering changing the name from bootstrap-sass-loader.
+Load Bootstrap styles and scripts in your Webpack bundle. This loader uses SASS to process CSS styles. Bootstrap 3 & 4 are supported.
+
+## Installation
+
+Get it via npm:
+
+```bash
+npm install bootstrap-loader
+```
+
+Don't forget to install these dependencies:
+
+```bash
+# Bootstrap 3
+npm install bootstrap-sass
+
+# or Bootstrap 4
+npm install twbs/bootstrap#v4.0.0-alpha
+
+# Node SASS & other loaders needed to handle styles
+npm install css-loader node-sass resolve-url-loader sass-loader style-loader
+```
+
+## Usage
+
+Simply require it:
+
+```js
+require('bootstrap-loader');
+```
+
+Or add `bootstrap-loader` as an entry point in your webpack config:
+
+```js
+entry: [ 'bootstrap-loader', './app' ]
+```
+
+Config is optional. It can be placed in root dir with name `.bootstraprc`. You can write it in `YAML` or `JSON` formats.
+
+```yaml
+---
+# You can use comments here
+useFlexbox: true
+
+styleLoaders:
+  - style
+  - css
+  - sass
+
+styles:
+  normalize: true
+  print: true
+
+scripts:
+  alert: true
+  button: true
+```
+
+```json
+{
+  // And JSON comments also!
+  "useFlexbox": true,
+
+  "styleLoaders": ["style", "css", "sass"],
+
+  "styles": {
+    "normalize": true,
+    "print": true
+  },
+
+  "scripts": {
+    "alert": true,
+    "button": true
+  }
+}
+```
+
+If no config provided, default one for Bootstrap 3 will be used.
+
+## Examples
+
+Check out example apps:
+
+* Basic usage: https://github.com/shakacode/bootstrap-loader-example
+* With CSS Modules: https://github.com/shakacode/bootstrap-loader-css-modules-example
+
+## Options
+
+Here are options for Bootstrap 3 & 4.
+
+### Bootstrap 3
+
+#### `loglevel`
+
+Default: `disabled`
+
+Outputs debugging info. In case you have troubles with loader you can set this option to `debug` and send us output form console.
```

Comment:
Actually this option is commented out by default, and in case if something is wrong — we are throwing. There is nothing to warn about, so if processing went smooth — output is clear. If we found some mistakes — they are thrown.